stacks/tcpconn.go

```diff
@@ -331,6 +334,8 @@ func (sock *TCPConn) checkPipeOpen() error {
 	return nil
 }
 
+//take a packet from the stack and and write it's payload into the socket's rx (ring) buffer
+// /\ i found this really misleading - the packet is not 'taken' from the stack, it is passed to the handler (called by PortStack RecvEth)
 func (sock *TCPConn) recv(pkt *TCPPacket) (err error) {
```

Comment:
See function documenting methodology for Go, also feel free to provide documentation changes as part of the PR, no need to discuss it in comments, the discussion can happen in comments here. More easy to follow.